CEF/cef/ingest/parser.yml

```
@@ -20,12 +20,49 @@ pipeline:
         input_field:  "{{parsed_event.message.end or parsed_event.message.endTime}}"
         output_field: date
 
+  # Parse 'rt': '1629701622409'
+  - name: parse_receipt_time_unix
+    filter: '{{parsed_event.message.rt  != null and parsed_event.message.rt  | re_match("\d{10,13}")}}'
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{parsed_event.message.rt | float / 1000}}"
+        output_field: date
+        format: 'timestamp'
+
+  # Parse 'rt': 'Feb 28 2021 18:20:54'
+  - name: parse_receipt_time_date
+    filter: '{{parsed_event.message.rt  != null and parse_receipt_time_unix==null}}'
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{parsed_event.message.rt }}"
+        output_field: date
```

The stage `Date` was update with a better support for timestamp date. I suggest to symplify this section.

```suggestion
  # Parse 'rt' (as a timestamp or as a date)
  - name: parse_receipt_time
    external:
      name: date.parse
      properties:
        input_field: "{{parsed_event.message.rt}}"
        output_field: date
```